contrib/clojure-package/test/test-symbol-random-api.clj

```diff
@@ -0,0 +1,118 @@
+(ns
```

Comment:
Same question - Do we need to have this file checked in? - it looks like it is written to during the generator test to compare contents

Reply:
same answer! Ooooops!